bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "iptools-jquery-modal",
   "description": "Multifunctional jQuery modal component",
   "main": "dist/iptools-jquery-modal.min.js",
-  "version": "0.1.3",
+  "version": "0.2.1",
```

Comment:
@beroes I think I mixed it up with mlhmenu by mentioning the version could be `0.2.1`. If last version was `0.1.3`, the breaking change should land in a version of `0.2.0`. Sorry for misleading you.